### ISSUE TITLE
My schematic won't load

### ISSUE THREAD

**Issue by charlieh0tel** [theacodes/kicanvas#100]
https://github.com/charlieh0tel/ee_stuff/blob/main/projects/rpi_sa8x8hat/rpi_sa8x8hat.kicad_sch

**Comment by charlieh0tel** [comment]
Perhaps something to do with hierarchy.

![image](https://github.com/user-attachments/assets/2e4b8342-3ca9-4e3d-a464-6c01a76bf423)

**Comment by devinatkin** [comment]
Similar issue for this one I  think https://github.com/Mr-Bossman/f1c200s-dev-board
I think more of a pain than the failure to load is simply the fact that it doesn't spit out any sort of hey I tried to load this and failed to get an output warning.

**Comment by theacodes** [comment]
Yeah, I would like to add some error handling, but I haven't had any time to work on this lately.
